README.md

```diff
@@ -1,288 +1,137 @@
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 world-info project](#-capstone-react)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 NBA-Information<a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**NBA-Information**
+NBA-population is a single-page website where users can view NBA Tems in the USA and can see their information e.g Confrence , Name, Division etc.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+    React.js
+    React , Reduc Toolkit
+    Javascript(ES6)
+    Webpack
+    Node.js
+    jest
+    Html/CSS
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+## Key Features <a name="key-features"></a>
 
-<!-- Features -->
+- **User can see total continent's countries**
+  **User can view every countinent countries detail's e.g population , language, timezones, region, area, native name**
 
-### Key Features <a name="key-features"></a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-> Describe between 1-3 key features of the application.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+-[Live Version](https://nba-info.onrender.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+## Loom Video
 
-## 🚀 Live Demo <a name="live-demo"></a>
+https://www.loom.com/share/a8864f72e1fa4788925e41ff4f9820c9
 
-> Add a link to your deployed project.
+## 💻 Getting Started <a name="getting-started"></a>
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- Create a local directory that you want to clone the repository.
```

Comment:
- [ ] ##### **Live version**

- [ ] It's one of the project requirements you deploy the application using Heroku, Netlify, or GitHub Pages. and make sure that:
1. The project is deployed properly, and the home and detail page works as expected. ✔️ 
2. A link to the deployed project is present in the README. ✔️ 
_ Let's address this 👍🏻